sdk/src/main/java/co/omise/android/models/ModelTypeResolver.kt

```diff
@@ -0,0 +1,48 @@
+package co.omise.android.models
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import com.fasterxml.jackson.databind.DatabindContext
+import com.fasterxml.jackson.databind.JavaType
+import com.fasterxml.jackson.databind.jsontype.impl.TypeIdResolverBase
+import java.util.*
```

Comment:
Could you use single import instead wildcard import?

Reply:
👍